lib/internal/simulation_manager_mixins/mtu_mixin.dart

```diff
@@ -0,0 +1,13 @@
+part of internal;
+
+mixin PeripheralMtuMixin on SimulationManagerBaseWithErrorChecks {
+
+  Future<int> requestMtuForDevice(String identifier, {int requestedMtu}) async {
```

Comment:
requestedMtu should not be optional

Reply:
but there is no sense to send MTU value, that is why it is optional.